### PR TITLE
community/events/ceph-days-nyc: Update talks and add lightning talk

### PR DESCRIPTION
**Pull request by Thingee**
Speaker cancellation.

Signed-off-by: Mike Perez <thingee@gmail.com>

### CODE REVIEW

**Comment by Thingee** [comment]
Whoops, I need to fix Anthony's speaker photo.

File: src/en/community/events/2023/ceph-days-nyc/index.md [hunk 1]
```diff
@@ -155,11 +155,11 @@ Benchmarking Ceph has always been a complex task - there are lots of tools but m
...
+Ceph enables flexible and scalable object storage of unstructured data for a wide variety of workloads. RGW (RADOS GateWay) deployments experience a wide variety of object sizes and must balance workload, cost, and performance requirements. S3 storage classes are an established way to steer data onto underlying media that meet specific resilience, cost, and performance requirements. One might for example define RGW back end storage classes for SSD or HDD media, non-redundant vs replicated vs erasure coding pools, etc. Diversion of individual objects or entire buckets into a non-default storage class usually requires specific client action. Compliance however can be awkward to request and impossible to enforce, especially in multi-tenant deployments that may include paying customers as well as internal users. This work enables the RGW back end to enforce storage class on uploaded objects based on specific criteria without requiring client actions. For example one might define a default storage class on performance TLC or Optane media for resource-intensive small S3 objects while assigning larger objects to Matt Vandermeulendense and cost-effective QLC SSD media.
```
Comment (anthonyeleven) [inline src/en/community/events/2023/ceph-days-nyc/index.md:160]: s/Matt Vandermeulen//

**Review by anthonyeleven — CHANGES_REQUESTED** [review]
Someone's name became inserted in the text last year, and it's still here, please remove.

File: src/en/community/events/2023/ceph-days-nyc/index.md (hunk 1 above)
Comment (Thingee) [reply src/en/community/events/2023/ceph-days-nyc/index.md]: Thanks, Anthony.

**Comment by Thingee** [comment]
Dan's photo has been added.

**Comment by Thingee** [comment]
If this is OK, please merge. There will be announcements going out this morning.

**Review by anthonyeleven — APPROVED** [review]

**Pull request merged**